robot/web/stream/camera/camera.go

```diff
@@ -27,7 +29,7 @@ func Camera(robot robot.Robot, stream gostream.Stream) (camera.Camera, error) {
 
 // VideoSourceFromCamera converts a camera resource into a gostream VideoSource.
 // This is useful for streaming video from a camera resource.
-func VideoSourceFromCamera(ctx context.Context, cam camera.Camera) gostream.VideoSource {
+func VideoSourceFromCamera(ctx context.Context, cam camera.Camera) (gostream.VideoSource, error) {
```

Comment:
Do we have a test case for when this returns an error?

Reply:
@hexbabe you can make an injected camera with bad bytes  to mimic y malformed camera,  in the test file for this package.

Reply:
Yeah can do. Incorporating that with the test Nick pointed out was missing.

Reply:
Added one